config/kernel-monitor.json

```diff
@@ -1,5 +1,7 @@
 {
 	"logPath": "/log/kern.log",
+	"lookback": "1h",
```

Comment:
looking back 1h might be too long to find panic information? We only care about the most recent, right? Changing it to 5m, or 10m maybe?